tests/integration/targets/elb_classic_lb/defaults/main.yml

```
@@ -161,3 +161,10 @@ default_logging_prefix: 'logs'
 updated_logging_prefix: 'mylogs'
 default_logging_interval: 5
 updated_logging_interval: 60
+
+local_certs:
+  - priv_key: "{{ remote_tmp_dir }}/private-1.pem"
+    cert: "{{ remote_tmp_dir }}/public-1.pem"
+    csr: "{{ remote_tmp_dir }}/csr-1.csr"
+    domain: "acm1.{{ resource_prefix }}.ansible.com"
```

```suggestion
    domain: "elb-classic.{{ tiny_prefix }}.ansible.test"
```

1) IIRC resource_prefix isn't guaranteed to be valid for domain names
2) It's possible that ACM will grumble about the use of `.test` but this is an example of what the `.test` domain is for.
3) When it comes to debugging why resources are getting left behind it's helpful for resources to include the name of what's being tested